Decred Wallet/Extensions/UIColor.swift

```diff
@@ -23,6 +23,9 @@ extension UIColor {
         static let offWhite = UIColor(hex:"#F3F5F6")
         static let lightOffWhite = UIColor(hex: "#F9FBFA")
         static let yellowWarning = UIColor.init(hex: "#FFC84E")
+        static let tabActive = UIColor.init(hex: "#4e5f70")
```

Comment:
Let's refrain from naming color constants after their usages at the moment. Use actual color names for constants e.g. `white`, `black`, `navyBlue` etc.

We should be able to use this color constant on a different object such as label and not necessarily tabs only.

Also, naming colors properly helps to prevent us from unnecessarily duplicating colors and allows us to merge/reuse very similar colors.